test-app/tests/integration/components/headless-form-native-validation-test.gts

```diff
@@ -472,6 +472,81 @@ module(
       assert.dom('[data-test-last-name-errors]').doesNotExist();
     });
 
+    test('opt out of native validation', async function (assert) {
```

Comment:
Suggestion: `'opt out of native validation'` -> `'opt out of native validation using @ignoreNativeValidation'`